src/Evolution/Executables/GeneralizedHarmonic/CMakeLists.txt

```diff
@@ -22,10 +21,24 @@ set(LIBS_TO_LINK
   Utilities
   )
 
-add_spectre_parallel_executable(
-  EvolveGeneralizedHarmonic
-  EvolveGeneralizedHarmonic
-  Evolution/Executables/GeneralizedHarmonic
-  EvolutionMetavars
-  "${LIBS_TO_LINK}"
+function(add_generalized_harmonic_executable
+  EXECUTABLE_NAME INITIAL_GUESS BOUNDARY_CONDITIONS)
+  add_spectre_parallel_executable(
+    "Evolve${EXECUTABLE_NAME}"
+    EvolveGeneralizedHarmonic
+    Evolution/Executables/GeneralizedHarmonic
+    "EvolutionMetavars<${INITIAL_GUESS}, ${BOUNDARY_CONDITIONS}>"
+    "${LIBS_TO_LINK}"
+    )
+endfunction(add_generalized_harmonic_executable)
+
+add_generalized_harmonic_executable(
+  KerrSchild
+  GeneralizedHarmonic::Solutions::WrappedGr<gr::Solutions::KerrSchild>
+  GeneralizedHarmonic::Solutions::WrappedGr<gr::Solutions::KerrSchild>
+)
+target_link_libraries(
+  EvolveKerrSchild
+  PRIVATE
```

Comment:
What does PRIVATE do here? I think I have the gist, that for this one executable, the target gets linked against GeneralRelativitySolutions...I'm just unclear what PRIVATE does.

Reply:
Since this is not a library I don't think it matters whether this is `PRIVATE` or `PUBLIC`. The call to `target_link_libraries` needs a keyword though, so I chose `PRIVATE`. In other words: Since `PUBLIC` is just `PRIVATE + INTERFACE` but `INTERFACE` has no meaning for an executable I think `PRIVATE` is technically the correct choice.

For libraries remember this: link libs with `PRIVATE` when they are used only in cpp files, link them with `INTERFACE` if they are used only in hpp files and link them with `PUBLIC` if they are used in both.